v22.1/deploy-app-vercel.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Deploy a Web App Built on CockroachDB with Vercel
+summary: Learn how to use Vercel and CockroachDB Serverless.
+toc: true
+twitter: false
+referral_id: docs_vercel
+docs_area: get_started
+---
+
+This tutorial shows you how to use [Vercel](https://vercel.com/) to deploy a web application built with [Next.js](https://nextjs.org/), [Express](https://expressjs.com/), [Prisma](https://www.prisma.io/), and {{ site.data.products.serverless }}.
+
+{{site.data.alerts.callout_success}}
+For more detailed instructions on building a fullstack app with Next.js and Prisma, see Vercel's official [How to Build a Fullstack App with Next.js, Prisma, and PostgreSQL](https://vercel.com/guides/nextjs-prisma-postgres) guide.
+{{site.data.alerts.end}}
+
+## Prerequisites
+
+Before starting the tutorial, do the following:
+
+1. Create a [{{ site.data.products.db }}](https://cockroachlabs.cloud/signup?referralId={{page.referral_id}}) account.
+
+1. Create a [Vercel](https://vercel.com/signup) account. You can do this with your GitHub login credentials.
+
+## Step 1. Create a {{ site.data.products.serverless }} cluster
+
+{% include cockroachcloud/quickstart/create-a-free-cluster.md %}
+
+<a name="connection-string"></a>
+
+After the cluster is created, the **Connection info** window appears. Click the **Connection string** tab and copy the connection string to a secure location. You will use this connection string to connect to CockroachDB later in the tutorial.
+
+{{site.data.alerts.callout_info}}
+The connection string is pre-populated with your username, cluster name, and other details, including your password. Your password, in particular, will be provided only once. Save it in a secure place (we recommend a password manager) to connect to your cluster in the future. If you forget your password, you can reset it by going to the [**SQL Users** page](../cockroachcloud/user-authorization.html).
+{{site.data.alerts.end}}
+
+## Step 2. Get the code
+
+1. Clone the `cockroachdb` fork of the `prisma-examples` repo:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ git clone git@github.com:cockroachdb/prisma-examples.git
+    ~~~
+
+    {{site.data.alerts.callout_info}}
+    In this tutorial, we deploy a slightly modified version of Prisma's [`rest-nextjs-api-routes`](https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-nextjs-api-routes) example app.
+    {{site.data.alerts.end}}
+
+1. Navigate to the `rest-nextjs-api-routes` directory:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    $ cd typescript/rest-nextjs-api-routes
```

Comment:
Do we need to have the project in a subdirectory? What would happen if we just put everything into the root of the project?